src/autoupdater.ts

```diff
@@ -3,14 +3,10 @@ import { GitHub } from '@actions/github/lib/utils';
 import * as ghCore from '@actions/core';
 import * as octokit from '@octokit/types';
 import { ConfigLoader } from './config-loader';
+import { Endpoints } from '@octokit/types';
 
-interface MergeOpts {
-  owner: string;
-  repo: string;
-  base: string;
-  head: string;
-  commit_message?: string;
-}
+type pullRequestResponse = Endpoints['GET /repos/{owner}/{repo}/pulls/{pull_number}']['response'];
+type mergeParameters = Endpoints['POST /repos/{owner}/{repo}/merges']['parameters'];
```

Comment:
It's better if types are PascalCase, it helps differentiate them from variables:
```suggestion
type PullRequestResponse = Endpoints['GET /repos/{owner}/{repo}/pulls/{pull_number}']['response'];
type MergeParameters = Endpoints['POST /repos/{owner}/{repo}/merges']['parameters'];
```